iothub/service/src/AmqpServiceClient.cs

```diff
@@ -12,6 +12,8 @@
 using Microsoft.Azure.Devices.Common;
 using Microsoft.Azure.Devices.Common.Data;
 using Microsoft.Azure.Devices.Common.Exceptions;
+using Microsoft.Azure.Devices.Shared;
+using AmqpTrace = Microsoft.Azure.Amqp.AmqpTrace;
```

Comment:
Was this alias necessary?

Reply:
Yeah, we have an "empty" `AmqpTrace` in our commond directory, which was creating a conflict: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/iothub/service/src/Common/AmqpTrace.cs

Reply:
Yikes. I know it would be pulling a type out, but is it worth evaluating whether we should do that? What would the negative impact be on a customer?

Reply:
Interestingly enough, it is an internal class. I say, yes, let's remove it.